composer.json

```diff
@@ -23,7 +23,7 @@
         "phpunit/phpunit": "^5.6",
         "sllh/php-cs-fixer-styleci-bridge": "^2.0",
         "symfony/form": "^2.7 || ^3.0",
-        "symfony/framework-bundle": "^2.7 || ^3.0",
+        "symfony/framework-bundle": "^2.7.8 || ^3.0",
```

Comment:
Why this constraint change?

Reply:
Because 2.7.0 - 2.7.7 failed to express a dependency on the sf/finder component they had, which made the build failon prefer-lowest: https://travis-ci.org/greg0ire/enum/jobs/288233304